<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cachingSettings</class>
 <widget class="QWidget" name="cachingSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="cacheModeLabel">
     <property name="text">
      <string>Cache Mode</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cacheModeComboBox">
     <item>
      <property name="text">
       <string>Look-Ahead</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No Caching</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="cacheSizeLabel">
     <property name="text">
      <string>Cache Size</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="cacheSizeDoubleSpinBox">
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="value">
      <double>50.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="readerThreadsLabel">
     <property name="text">
      <string>Reader Threads</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="readerThreadsSpinBox">
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
